bin/start.js

```diff
@@ -35,16 +35,6 @@ browserSync.init({
             ],
             fn: () => {
                 Build.buildSync();
-
-                /**
-                 * Always ensure that this task runs last, as it needs to wait for all
-                 * synchronous build events to complete so that browserSync can finish
-                 * and reload the page (and therefore, allow selenium to connnect).
-                 */
-                const tests = exec('yarn run test:regression -u', { cwd: CONFIG.testDir });
-                // this *is* asynchronous, see comment above
-                tests.stdout.pipe(process.stdout);
-                tests.stderr.pipe(process.stderr);
```

Comment:
@Droogans: This was removed because the process almost locked up my machine when moving files around while the dev server was running.

Reply:
Spent some time looking closer at this and I'm pretty sure this isn't going to fix the problem completely.

I've noticed that each file change event triggers these actions, meaning moving 10 files around will fire off 10 simultaneous build tasks. I'll look at limiting it globally with some sort of debounce or throttle.

Reply:
https://browsersync.io/docs/options#option-reloadDebounce
https://browsersync.io/docs/options#option-reloadThrottle